### DISCUSSION TITLE
[Unstable] Shields being one-handed

### DISCUSSION THREAD

**Discussion by PokkeTigrex** [FakeFishGames/Barotrauma#10174] [Unstable feedback]
Since the 0.20.0.0 is adding a new craftable shield, I feel shields should find more use outside of the usual annoying pushing people around that people usually only use it for. To do this, I think making shields one-handed could give it so much more uses, as it allows the use of one handed weapons along side the shield like revolvers, batons and swords to name some. The riot shield also could be a separate shield type from two handed moloch shield and act as a lighter, smaller shield that could be stored in the toolbelt/back slot (or hotbar if you want).
Really good example image of it in, with a whole XML for it. #10165.

![image](https://user-images.githubusercontent.com/85914375/198414555-95f61e2a-f8bb-490e-8fe9-8d42395887fa.png)

**Comment by Hazmat616** [comment]
I've been working on making some weapons one handed in mods and realized something intriguing. Any item that only uses one hand has the distinction righthand,lefthand. That simply means it can be in either. Right being the foremost and left rearmost. So some of the bulkier items you could still theoretically hold in one hand but not be able to hold two of could simply only have one of those distinctions. This would mean you could make riot shields be able to pair with a one handed weapon while taking up a specific hand. Ideally the same as other problematic items you also make one handed.
